Assignments/Week2/add_binary_strings.py

```diff
@@ -0,0 +1,8 @@
+def addBinary(a, b):
+        s=bin(int(a,2)+int(b,2))
```

Comment:
This is definitely not a good way to solve this question. You won't get this library function in any other language. This how you can solve this by simple programming logic.